package.json

```diff
@@ -36,16 +36,24 @@
       "name": "Michael Prentice",
       "email": "michael@xlts.dev",
       "url": "https://github.com/splaktar"
+    },
+    {
+      "name": "Jaroslaw Zolnowski",
+      "email": "jarek@xlts.dev",
+      "url": "https://github.com/jzolnowski"
     }
   ],
-  "main": "index.js",
+  "main": "main.js",
   "scripts": {
-    "start": "webpack-dev-server --open --mode development",
+    "pre-commit": "lint-staged && npm run lint",
+    "build:templates": "gulp templates",
+    "start": "npm run build:templates && webpack-dev-server --open --mode development --config config/webpack.dev.mjs",
```

Comment:
I just realized that this command will not pick up template changes while it is running. I suggest we use something like [npm-run-all][1] to run `gulp watch` and `webpack-dev-server` in parallel.

[1]: https://www.npmjs.com/package/npm-run-all